vernac/comFixpoint.ml

```diff
@@ -172,6 +172,8 @@ type ('constr, 'types) recursive_preentry =
 
 let fix_proto sigma =
   Evd.fresh_global (Global.env ()) sigma (Coqlib.lib_ref "program.tactic.fix_proto")
+let fix_proto_relevance = Sorts.Relevant
+(* Would probably be overkill to use a specific fix_proto in SProp when in SProp?? *)
```

Comment:
We could make fix_proto sort poly
IDK what it's used for TBH

Reply:
> We could make fix_proto sort poly

Ah, right. But probably not very prioritary.

> IDK what it's used for TBH

Me neither TBH. I suspect it is used to control the reduction or something of that sort.